### ISSUE TITLE
DragPanelExtender - JavaScript errors in IE 11 with non-100% zoom

### ISSUE THREAD

**Issue by VKB2612** [DevExpress/AjaxControlToolkit#522]
Hi,

The DragPanelExtender does not drag correctly in all browsers. 

IE v11 -> throws JS error
Firefox -> after dragging, the panel relocates to its old position
Chrome -> after dragging, the panel relocates to its old position
Edge -> after dragging, the panel relocates to its old position

### The installation method
- [x ] Installer
- [ ] NuGet package
- [ ] A custom build from the source code

### Minimal steps to reproduce the bug
For IE v11 -> Run the demo page (https://ajaxcontroltoolkit.devexpress.com/DragPanel/DragPanel.aspx) and perform drag 

For other browsers, please refer the code shared below

### Actual result
Drag functionality fails

### Expected result
Drag functionality should work correctly in all browsers

### Browser(s) used
IE v11
Firefox (v68.4.1)
Chrome (v79.0.3945.130)
Edge (v44.17763.831.0)

### A site deployment method
- [ ] VS development web-server,
- [x ] IIS

As I am not able to upload project files or video showing error, the code is shared below.

aspx:

        <asp:Panel ID="Dialog_Panel" runat="server">
            <div class="StatisticForm">
                <asp:Table ID="StatisticTable" Style="width: 100%" CellPadding="0" CellSpacing="1" BorderWidth="1" HorizontalAlign="Center" runat="server">
                    <asp:TableRow>
                        <asp:TableCell>
                            <asp:Panel ID="Content_DemoContent_Panel6" runat="server">
                                <div id="Content_DemoContent_Panel7" style="border-color: Black; border-width: 2px; border-style: Solid; height: 20px; width: 100%;">
                                    <div class="dragMe">Drag Me</div>
                                </div>
                                <div id="Content_DemoContent_Panel8" style="color: WhiteSmoke; background-color: #0B3D73; border-color: Black; border-width: 2px; border-style: Solid; height: 250px; width: 100%; overflow: scroll;">
                                    <div>
                                        <p>This panel will reset its position on a postback or page refresh.</p>
                                        <hr />
                                        <p>ASP.NET AJAX is a free framework for building a new generation of richer, more interactive, highly personalized cross-browser web applications.  This new web development technology from Microsoft integrates cross-browser client script libraries with the ASP.NET 2.0 server-based development framework.  In addition, ASP.NET AJAX offers you the same type of development platform for client-based web pages that ASP.NET offers for server-based pages.  And because ASP.NET AJAX is an extension of ASP.NET, it is fully integrated with server-based services. ASP.NET AJAX makes it possible to easily take advantage of AJAX techniques on the web and enables you to create ASP.NET pages with a rich, responsive UI and server communication.  However, AJAX isn't just for ASP.NET.  You can take advantage of the rich client framework to easily build client-centric web applications that integrate with any backend data provider and run on most modern browsers.  </p>
                                    </div>
                                </div>
                            </asp:Panel>
                            <ajaxToolkit:DragPanelExtender
                                ID="DragPanelExtenderDragMePanel"
                                runat="server"
                                TargetControlID="Content_DemoContent_Panel6"
                                DragHandleID="Content_DemoContent_Panel6" />
                        </asp:TableCell>
                    </asp:TableRow>
                </asp:Table>
            </div>
        </asp:Panel>
  

Please check this.
Thanks and regards

**Comment by AlekseyMartynov** [comment]
Hello

> IE v11 -> throws JS error

I don't see any errors in IE 11:
![ie11](https://user-images.githubusercontent.com/4426185/73924769-175afd00-48de-11ea-9804-700e0bccca2a.png)

> after dragging, the panel relocates to its old position

Make sure that you don't remove this script block:

https://github.com/DevExpress/AjaxControlToolkit/blob/0a365c6fcf34c465c5ffb5bf2b1577debc6901ce/AjaxControlToolkit.SampleSite/DragPanel/DragPanel.aspx#L7-L17

**Comment by VKB2612** [comment]
Hi @AlekseyMartynov,
Thanks for your reply. I still see the same error in IE 11 (exact: 11.973.17763.0). Did you check the code shared above to test in other browsers? I have prepared a short video showing the issue but I am not able to upload it here. It always throws error "Something went really wrong, and we can't process that file" no matter what file type is. Is there any other way to share the files? 
Regards

**Comment by AlekseyMartynov** [comment]
> Did you check the code shared above to test in other browsers?

Yes. Refer to the second half of my answer.

> Is there any other way to share the files?

- Place the files in a ZIP archive and upload here.
- Open a ticket at https://devexpress.com/sc. Please include a reference to this thread.

**Comment by VKB2612** [comment]
Hi,

I have created the ticket. Please check. [https://supportcenter.devexpress.com/ticket/details/t860243/dragpanelextender-does-not-work-correctly](https://supportcenter.devexpress.com/ticket/details/t860243/dragpanelextender-does-not-work-correctly)
Second half of the answer did not make any difference.

**Comment by AlekseyMartynov** [comment]
Thank you.

IE error details:
![ie-error](https://user-images.githubusercontent.com/4426185/74019032-f8c13880-49a7-11ea-9f49-4f3fa456c1bc.png)

Corresponding source:
```js
// Name:        MicrosoftAjax.debug.js
// Assembly:    System.Web.Extensions
// Version:     4.0.0.0
// FileVersion: 4.7.3535.0
```
```js
Sys.UI.DomElement.setLocation = function Sys$UI$DomElement$setLocation(element, x, y) {
    /// <summary locid="M:J#Sys.UI.DomElement.setLocation" />
    /// <param name="element" domElement="true"></param>
    /// <param name="x" type="Number" integer="true"></param>
    /// <param name="y" type="Number" integer="true"></param>
    var e = Function._validateParams(arguments, [
        {name: "element", domElement: true},
        {name: "x", type: Number, integer: true},
        {name: "y", type: Number, integer: true}
    ]);
    if (e) throw e; // line 4496
    var style = element.style;
    style.position = 'absolute';
    style.left = x + "px";
    style.top = y + "px";
}
```

**Comment by AlekseyMartynov** [comment]
I'm attaching a corrected version of `DragExtender_Test.aspx`:
[DragExtender_Test.aspx.zip](https://github.com/DevExpress/AjaxControlToolkit/files/4170366/DragExtender_Test.aspx.zip)

IE 11 errors occur with non-100% zoom. We'll fix it. Meanwhile, add the following script block to the bottom of the page:
```html
<script>
    var orig_MSAjaxSetLocation = Sys.UI.DomElement.setLocation;
    Sys.UI.DomElement.setLocation = function (element, x, y) {
        return orig_MSAjaxSetLocation(element, Math.round(x), Math.round(y));
    }
</script>
```
